app/assets/javascripts/spree/frontend/add_recently_viewed_products.js.coffee

```diff
@@ -0,0 +1,5 @@
+Spree.fetch_recently_viewed_products = (productId = "")->
+  $.ajax
+    url: Spree.pathFor("recently_viewed_products?product_id=" + productId),
```

Comment:
@damianlegawiec Not sure why are we sending productID here if we plan to fetch all recently viewed products.
I think I am missing something here.